cmd/following.go

```diff
@@ -0,0 +1,42 @@
+package cmd
```

Comment:
Move this command under list command. Check followers command for reference

Reply:
Done